Kernel/Devices/SerialDevice.h

```diff
@@ -112,25 +112,25 @@ class SerialDevice final : public CharacterDevice {
     virtual const char* class_name() const override { return "SerialDevice"; }
 
     void initialize();
-    void set_interrupts(char interrupt_enable);
+    void set_interrupts(bool interrupt_enable);
     void set_baud(Baud);
-    void set_fifo_control(char fifo_control);
+    void set_fifo_control(u8 fifo_control);
     void set_line_control(ParitySelect, StopBits, WordLength);
     void set_break_enable(bool break_enable);
-    void set_modem_control(char modem_control);
-    char get_line_status() const;
+    void set_modem_control(u8 modem_control);
+    u8 get_line_status() const;
     bool rx_ready();
     bool tx_ready();
 
-    int m_base_addr;
-    char m_interrupt_enable;
-    char m_fifo_control;
+    u32 m_base_addr;
+    bool m_interrupt_enable;
+    u8 m_fifo_control;
```

Comment:
Can we also explicitly initialize primitives while we're at it? :^)

Reply:
sure :)